libkatana_python_native/include/katana/python/Conventions.h

```diff
@@ -241,7 +237,7 @@ GetRangeStep(T& self) -> std::remove_reference_t<decltype(*self.begin())> {
     return 1;
   }
   auto next = self.begin();
-  next++;
+  std::advance(next, 1);
```

Comment:
What's the need for this method as opposed to just incrementing the iterator?

Reply:
It's just the more STL correct way to do it. I think there are some esoteric iterators that support `advance` but not `++`. I think it's for things that should appear as iterators even though they don't natively support the concept. This was just a clean up.